tools/clang/test/CodeGenSPIRV/spirv.debug.opline.precedence.hlsl

```diff
@@ -0,0 +1,28 @@
+// Run: %dxc -T cs_6_0 -E main -Zi
+
+// CHECK:      [[file:%\d+]] = OpString
+// CHECK-SAME: spirv.debug.opline.precedence.hlsl
+
+void main() {
+  int a;
+  int b;
+
+//CHECK:      OpLine [[file]] 13 3
+//CHECK-NEXT: OpSelectionMerge %switch_merge None
+  switch (a) {
+  default:
+    b = 0;
+  case 1:
+    b = 1;
+    break;
+  case 2:
+    b = 2;
+  }
+
+// TODO: revisit line info for control-flow
+//CHECK:      OpLine [[file]] 26 23
```

Comment:
the merge point for the loop is probably line 27 in the HLSL, which is line 28 in the preprocessed file. So I would think that `26` should in fact be `28`? Is this why you put a `TODO` here?

Reply:
This can be addressed in a different PR.

Reply:
Never mind. I just removed the TODO.